src/SFML/Window/Win32/CursorImpl.hpp

```diff
@@ -92,8 +90,8 @@ class CursorImpl : NonCopyable
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    HCURSOR m_cursor;
-    bool m_systemCursor;
+    void* m_cursor;
```

Comment:
Might want to add a comment to justify the usage of a void* here, so it's not removed in the next refactoring

Reply:
Will do